web_notify/models/res_users.py

```diff
@@ -32,28 +32,43 @@ def _compute_channel_names(self):
     notify_info_channel_name = fields.Char(compute="_compute_channel_names")
     notify_default_channel_name = fields.Char(compute="_compute_channel_names")
 
-    def notify_success(self, message="Default message", title=None, sticky=False):
+    def notify_success(
+        self, message="Default message", title=None, sticky=False, params=None
+    ):
         title = title or _("Success")
-        self._notify_channel(SUCCESS, message, title, sticky)
+        self._notify_channel(SUCCESS, message, title, sticky, params)
 
-    def notify_danger(self, message="Default message", title=None, sticky=False):
+    def notify_danger(
+        self, message="Default message", title=None, sticky=False, params=None
+    ):
         title = title or _("Danger")
-        self._notify_channel(DANGER, message, title, sticky)
+        self._notify_channel(DANGER, message, title, sticky, params)
 
-    def notify_warning(self, message="Default message", title=None, sticky=False):
+    def notify_warning(
+        self, message="Default message", title=None, sticky=False, params=None
+    ):
         title = title or _("Warning")
-        self._notify_channel(WARNING, message, title, sticky)
+        self._notify_channel(WARNING, message, title, sticky, params)
 
-    def notify_info(self, message="Default message", title=None, sticky=False):
+    def notify_info(
+        self, message="Default message", title=None, sticky=False, params=None
+    ):
         title = title or _("Information")
-        self._notify_channel(INFO, message, title, sticky)
+        self._notify_channel(INFO, message, title, sticky, params)
 
-    def notify_default(self, message="Default message", title=None, sticky=False):
+    def notify_default(
+        self, message="Default message", title=None, sticky=False, params=None
+    ):
         title = title or _("Default")
-        self._notify_channel(DEFAULT, message, title, sticky)
+        self._notify_channel(DEFAULT, message, title, sticky, params)
 
     def _notify_channel(
-        self, type_message=DEFAULT, message=DEFAULT_MESSAGE, title=None, sticky=False
+        self,
+        type_message=DEFAULT,
+        message=DEFAULT_MESSAGE,
+        title=None,
+        sticky=False,
+        params=None,
```

Comment:
```suggestion
        **params,
```
No? This is already a dict, no need to cast to a dict afterwards.
not blocking